src/jvmMain/kotlin/io/beatmaps/api/review.kt

```diff
@@ -453,7 +453,8 @@ fun Route.reviewRoute() {
             val response = requireCaptcha(
                 reply.captcha,
                 {
-                    newSuspendedTransaction {
+                    var insertedId: Int? = null
+                    val response = newSuspendedTransaction {
```

Comment:
val (insertedId, response) = newSuspendedTransaction {

Reply:
I tried that initially, but overlooked something. Looking at it again I found my issue. Much better!